aten/src/ATen/cpu/vec/vec256/vsx/vec256_int64_vsx.h

```diff
@@ -1,8 +1,8 @@
 #pragma once
 
-#include <ATen/cpu/vec/vec256/intrinsics.h>
-#include <ATen/cpu/vec/vec256/vec256_base.h>
-#include <ATen/cpu/vec/vec256/vsx/vsx_helpers.h>
+#include <ATen/cpu/vec/Vectorized/intrinsics.h>
+#include <ATen/cpu/vec/Vectorized/Vectorized_base.h>
+#include <ATen/cpu/vec/Vectorized/vsx/vsx_helpers.h>
```

Comment:
Huh, what's going on here?

Reply:
Ouch sorry. I made that patch with 1.8.1 and ported it to master where the vec256 class was renamed but not the paths (which I find confusing) will fix this.